docs-devsite/vertexai-preview.md

```diff
@@ -367,3 +371,27 @@ export declare enum HarmSeverity
 |  HARM\_SEVERITY\_NEGLIGIBLE | <code>&quot;HARM_SEVERITY_NEGLIGIBLE&quot;</code> |  |
 |  HARM\_SEVERITY\_UNSPECIFIED | <code>&quot;HARM_SEVERITY_UNSPECIFIED&quot;</code> |  |
 
+## VertexAIErrorCode
+
+Standardized error codes that [VertexAIError](./vertexai-preview.vertexaierror.md#vertexaierror_class) can have.
+
+<b>Signature:</b>
+
+```typescript
+export declare const enum VertexAIErrorCode 
+```
+
+## Enumeration Members
+
+|  Member | Value | Description |
+|  --- | --- | --- |
+|  ERROR | <code>&quot;error&quot;</code> | A generic error occured. |
+|  FETCH\_ERROR | <code>&quot;fetch-error&quot;</code> | An error occurred while performing a fetch. |
+|  INVALID\_CONTENT | <code>&quot;invalid-content&quot;</code> | An error associated with a Content object. |
+|  NO\_API\_KEY | <code>&quot;no-api-key&quot;</code> | An error occured due to a missing api key. |
```

Comment:
Is this specific to the "Firebase API key"?  If so, then can we say that?

`An error occurred due to a missing Firebase API key.`

Reply:
Yeah, it's specific to the Firebase API key. I changed the comment to `An error occurred due to a missing Firebase API key.`.